src/dos/dos_programs.cpp

```diff
@@ -1282,7 +1294,26 @@ class IMGMOUNT : public Program {
 
 		// find all file parameters, assuming that all option parameters have been removed
 		while (cmd->FindCommand((unsigned int)(paths.size() + 2), temp_line) && temp_line.size()) {
+			// Try to find the path on native filesystem first
+			const std::string real_path = to_native_path(temp_line);
```

Comment:
The above chunk of file patch check code is similar (or identical) to this chunk; can they be made into a collapsed helper function? `FileExistsInLocalOrHome` or  `FileExistsRelativeOrInHome`? (Or something to that effect).
Just a though; not critical as the code is relatively small.

Reply:
I'm not a fan of this idea - mostly because it would be awkward to make the code common (logs would need to be parametrized…). Even suggested file names are weird.

Long-term, I think the bigger chunk of code could be maybe split off from IMGMOUNT and MOUNG and shared, but as for now - I prefer to keep it as it is.